index.html

```diff
@@ -144,49 +144,33 @@ <h2 class="title">Type of bicycles</h2>
-                                    All of the stages are timed to the finish; the riders' times are compounded
-                                    with their previous stage times. The rider with the lowest cumulative
-                                    finishing times is the leader of the race and wears the yellow jersey.
-                                </p>
-                                <p>
-                                    While the general classification garners the most attention, there are
-                                    other contests held within the Tour: the points classification for
-                                    the sprinters, the mountains classification for the climbers, young
-                                    rider classification for riders under the age of 26, and the team classification
-                                    for the fastest teams. Achieving a stage win also provides prestige,
-                                    often accomplished by a team's cycling sprinter specialist.
-                                </p>
-                                <a class="big-button">Read more</a>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+        <section class="cycle-service">
+            <div class="cycle-col">
+                <img src="images/competition/logo.png" class="cycle-col__image" alt="Tour de france">
+            </div>
+            <div class="cycle-col">
+                <h3 class="cycle-col__title">Tour de france</h3>
+                <p class="cycle-col__text">
+                    The Tour de France is an annual men's multiple stage bicycle race
+                    primarily held in France, while also occasionally passing through
+                    nearby countries. Like the other Grand Tours (the Giro d'Italia and
+                    the Vuelta a España), it consists of 21 day-long stages over the course of 23 days.
+                </p>
+                <p class="cycle-col__text">
+                    The race was first organized in 1903 to increase sales for the newspaper L'Auto
+                    and is currently run by the Amaury Sport Organisation. The race has been held
+                    annually since its first edition in 1903 except when it was stopped for the two
+                    World Wars.
+                </p>
+                <p class="cycle-col__text">
+                    There are usually between 20 and 22 teams, with eight riders in each.
+                    All of the stages are timed to the finish; the riders' times are compounded
+                    with their previous stage times. The rider with the lowest cumulative
+                    finishing times is the leader of the race and wears the yellow jersey.
+                </p>
+                <a href="#" class="cycle-col__link">
+                    <button class="cycle-col__link_button">Read more</button>
```

Comment:
is `_button` modifier? then it should be separated by double dashes `--`

Reply:
okay, problem with name of class resolved